cmd/marker/main.go

```diff
@@ -16,27 +16,38 @@ package main
 
 import (
 	"flag"
-	"time"
-
+	"github.com/Rican7/retry/jitter"
 	"github.com/golang/glog"
 	"github.com/kubevirt/bridge-marker/pkg/marker"
+	"time"
 )
 
 func main() {
 	nodeName := flag.String("node-name", "", "name of kubernetes node")
-	pollInterval := flag.Int("poll-interval", 10, "interval between updates in seconds, 10 by default")
+	updateInterval := flag.Int("update-interval", 60, "interval between updates in seconds, 60 by default")
+	pollInterval := flag.Int("poll-interval", 10, "interval between node info polls. every 10 cache updates by default")
 
 	flag.Parse()
 
 	if *nodeName == "" {
 		glog.Fatal("node-name must be set")
 	}
 
-	for {
-		err := marker.Update(*nodeName)
-		if err != nil {
-			glog.Errorf("Update failed: %v", err)
+	for i := 0; i <= *pollInterval; i++ {
+		useCache := true
+		if i == *pollInterval {
+			err := marker.Update(*nodeName, !useCache)
+			if err != nil {
+				glog.Errorf("Update failed: %v", err)
+			}
+		} else {
+			err := marker.Update(*nodeName, useCache)
+			if err != nil {
+				glog.Errorf("Update failed: %v", err)
+			}
+			// will use a default random generator
```

Comment:
I think this comment is redundant. Although it points to a question - is this using crypto library for random numbers? If not, it may mean that all instances of the marker will get the same jittered numbers. Would you please check it?